src/components/pages/Profile.js

```diff
@@ -1,12 +1,39 @@
-import React from "react";
+import React, {useContext} from "react";
 import NavBar from "../Navbar.js";
-const Profile = () => {
+import {SignIn} from "../auth/SignIn"
+
+
+export function Profile(){
+    const {userData, setUserData} = useContext(UserContext)
+  
   return (
     <div>
-      <NavBar />
-      <h3>User Profile goes here!</h3>
-      <p>Welcome user! Blah blah blah</p>
+      <NavBar/>
+      <h3>Welcome</h3>
+      <div>
```

Comment:
So, right here, you could leave comments like "// upload image button component could go here. Or upload image button could go here